packages/slice-machine/src/features/onboarding/content.tsx

```diff
@@ -8,92 +8,106 @@ import type {
 
 export const onboardingSteps: OnboardingStep[] = [
   {
-    id: "addSlice",
-    title: "Add Slices",
-    description: "Add slices to your page type.",
+    id: "createPageType",
+    title: "Create a Page Type",
+    description: "Build the structure for your page.",
   },
   {
     id: "codePage",
-    title: "Code your page",
-    description: "Build your page's structure",
+    title: "Code Your Page Type",
+    description: "Prepare your code to fetch content.",
   },
   {
-    id: "pushModels",
-    title: "Push Your Page",
-    description: "Push to your Page Builder.",
+    id: "createSlice",
+    title: "Create a Slice",
+    description: "Build a reusable website section.",
+  },
+  {
+    id: "reviewAndPush",
+    title: "Review and Push Changes",
+    description: "Enable editors to create content.",
   },
   {
     id: "createContent",
-    title: "Create content",
-    description: "Create engaging content.",
+    title: "Create Content in the Page Builder",
```

Comment:
I think there were some more recent updates to the copy

Reply:
Yeah I'm on it